release-notes/opensearch-common-utils.release-notes-2.19.0.0.md

```diff
@@ -0,0 +1,19 @@
+## Version 2.19.0.0 2025-02-03
+
+Compatible with OpenSearch 2.19.0
+
+### Maintenance
+* Increment version to 2.19.0-SNAPSHOT ([#749](https://github.com/opensearch-project/common-utils/pull/749))
+* Remove version workflow ([#455](https://github.com/opensearch-project/common-utils/pull/455))
+* Updates sample cert and admin keystore ([#598](https://github.com/opensearch-project/common-utils/pull/598))
+* Fix 2.x branch github CI workflow. ([#777](https://github.com/opensearch-project/common-utils/pull/777))
+
+### Bug Fixes
+* Fix bucket selector aggregation writeable name. ([#773](https://github.com/opensearch-project/common-utils/pull/773))
+
+### Refactoring
+* add should_create_single_alert_for_findings field to security-analytics ([#757](https://github.com/opensearch-project/common-utils/pull/757))
+* Monitor model changed to add an optional fanoutEnabled field ([#758](https://github.com/opensearch-project/common-utils/pull/758))
+
+### Documentation
+* Added 2.1.0.0 release notes. ([#780](https://github.com/opensearch-project/common-utils/pull/780))
```

Comment:
This is wrong? should be 2.19.0.0?

Reply:
This is a PR created by a bot, so I can't edit it directly. Will fix this typo in a subsequent PR.